code/dqe/symfilecheck.q

```diff
@@ -0,0 +1,8 @@
+\d .dqe
+hdbdir:@[value;`hdbdir;`hdb]
+symfilecheck:{[directory;filename]                     // if sym file exists
+  (c;"sym file named ",(","sv string(), filename)," ",$[c:.os.Fex .Q.dd[directory]filename;"exists";"doesn't exists"])
```

Comment:
Why is string stitching needed here? My assumption would be that filename is an atom.